src/System.IO.Pipes/tests/Interop.Windows.cs

```diff
@@ -14,17 +14,41 @@ namespace System.IO.Pipes.Tests
     /// </summary>
     internal static partial class Interop
     {
-        [DllImport("kernel32.dll", SetLastError = true)]
+        private const string Kernel32 = "kernel32.dll";
+
+        private const int ERROR_SUCCESS = 0x0;
+        private const int ERROR_CANNOT_IMPERSONATE = 0x558;
+
+        private const int LOAD_LIBRARY_SEARCH_SYSTEM32 = 0x00000800;
+
+        private const int CREDUI_MAX_USERNAME_LENGTH = 513;
+
+        sealed internal class SafeLibraryHandle : SafeHandleZeroOrMinusOneIsInvalid
+        {
+            [DllImport(Kernel32, CharSet = CharSet.Unicode, SetLastError = true, ExactSpelling = true)]
+            private static extern bool FreeLibrary(IntPtr hModule);
+
+            internal SafeLibraryHandle() : base(true) { }
+
+            override protected bool ReleaseHandle()
+            {
+                return FreeLibrary(handle);
+            }
+        }
+
+        [DllImport(Kernel32, EntryPoint = "LoadLibraryExW", CharSet = CharSet.Unicode, SetLastError = true)]
+        internal static extern SafeLibraryHandle LoadLibraryEx(string libFilename, IntPtr reserved, int flags);
```

Comment:
Is there a reason you can't include the actual common interop files instead? Rather not be duplicating unnecessarily.

Reply:
Thanks for pointing it out. Correct, we should be using the existing Common Interop files. I added the new iteration that consumes those now. Cross fingers so CI passes.